src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
+
+          <p> A <src>with-child-controls: no</src> directive on an <src>include-controls</src>
+            indicates that ONLY the explicitly matching control is explicitly included, any
+            descendants are not included. </p>
+
+          <p> If no <src>with-child-controls</src> is provided, the processor MUST consider the
+            directive as being equivalent to one having <src>with-child-controls:no</src>. </p>
+
+        </section>
+
         <section id="exclude-directive">
-          <head>Excluding controls</head>
+          <head>exclude-controls</head>
           <p>Exclusions work the same way as inclusions, except with the opposite effect - the
             indicated control(s) do not appear in the target catalog.</p>
-          <p>Additionally, there is no such thing as <src>exclude-all</src>, which is invalid and
-            should be considered inoperable.</p>
-          <p>Any control designated to be both included and excluded, is excluded. This holds
+          <p>Any control designated to be both included and excluded, MUST be excluded. This holds
             irrespective of the specificity of the selection for inclusion or exclusion. For
             example, if AC-1 is included by id <code>ac-1</code> and excluded by matching
               <code>ac.*</code>, it is excluded. Selection for exclusion prevails.</p>
-          <p><propose>One possible exception: what about when a control is excluded, but it has
-              descendant controls that are included? Under <src>include-all</src>, the descendants
-              maybe would be excluded. If the subcontrol is included by <src>include-controls</src>
-              perhaps the 'shells' of its ancestors should be included despite their nominal
-              exclusion. See section on "implicit inclusions under as-is", below.</propose></p>
+          <p><src>exclude-controls</src> will also have at least one <src>with-ids</src> and
+              <src>matching</src> directive, the processor MUST follow the same rules as defined in
+            the above sections for these directives, but exclude instead of include any
+            controls.</p>
         </section>
-        <section id="controls-order">
-          <head>Order of controls returned in selection</head>
-          <p>Because controls may be organized in the subsequence merge phase, order of controls
-            returned in the selection step is unimportant in most circumstances. If no merge
-            directive is applied, however, or the <q>no merge</q> combination rule is used (see
-            below), controls are returned for an import in the order of their appearance in their
-            catalog of origin.</p>
-          <p><revisit>BR suggests we revisit as an alternative would be to sort in call order. I
-              think the given proposal is simpler given cases of match and "all".</revisit></p>
+        <section id="redundant-calls">
```

Comment:
This is orphaned from the import section. This should be moved to be alongside these other sections.